Makefile

```diff
@@ -21,6 +21,8 @@ VENDOR := "SKB Kontur"
 URL := "https://github.com/moira-alert/moira"
 LICENSE := "MIT"
 
+services := "notifier" "api" "checker" "cli"
```

Comment:
Let's rename it as `SERVICES` or better `MAIN_TARGETS`

Reply:
fix it

Reply:
Corrected

Reply:
corrected